frontend/__tests__/inquiry/inquiries_table.test.tsx

```diff
@@ -139,24 +139,24 @@ describe("Inquiries Table", () => {
     }
   })
 
-  it("should log the inquiry details on console when clicked.", () => {
-    ;(useQuery as jest.Mock).mockReturnValue({
-      data: { data: singleInquiry },
-      isPending: false,
-    })
-    renderComponent()
-
-    console.log = jest.fn()
-
-    const inquiryRow = screen.getByTestId("inquiry-row")
-    inquiryRow.click()
-
-    expect(console.log).toHaveBeenCalledWith(
-      expect.objectContaining({
-        id: singleInquiry[0].id,
-        text: singleInquiry[0].text,
-        created_at: singleInquiry[0].created_at,
-      }),
-    )
-  })
+  // it("should log the inquiry details on console when clicked.", () => {
```

Comment:
@ilhanbae / @DmitriyMalayev 
Should this test be commented out?

Reply:
I added this test to meet the requirement of my task and was intended to be removed before production - we wouldn't want to log this detail in the production. Thinking about it, I think it was a mistake to put this test - logging this detail was not a requirement of the user story, but rather it was a tool for devs.

@DmitriyMalayev If you could, please remove the test you commented out.

Reply:
will do, thanks